tests/cyclient/scan_config/test_dev_scan_config.py

```diff
@@ -8,9 +8,16 @@ def test_get_service_name() -> None:
     assert dev_scan_config.get_service_name('iac') == '5026'
     assert dev_scan_config.get_service_name('sca') == '5004'
     assert dev_scan_config.get_service_name('sast') == '5004'
+    assert dev_scan_config.get_service_name('secret', True) == '5004'
```

Comment:
please use a named argument for bool because it's not clear what it mean

instead of `dev_scan_config.get_service_name('secret', True)` use
`dev_scan_config.get_service_name('secret', should_use_scan_service=True)`